helm-charts/sep-service/example_values.yaml

```diff
@@ -11,6 +11,9 @@ image:
    tag: latest
    pullPolicy: Always
 deployment:
+   args:
+       - '"--sep-server"'
+       - '"--stellar-observer"'
```

Comment:
❓  Do we need both the single `'` and double `"` quotes?

Reply:
Yes, unfortunately we do because the templating rules are a little particular. The short explanation is that we want to keep the double quotes around each element when we slot it into the deployment.